roles/openshift_repos/vars/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 known_openshift_deployment_types: ['origin', 'online', 'enterprise']
+known_openshift_product_types: ['openshift', 'atomic-enterprise']
```

Comment:
What is the origin equivalent to atomic-enterprise, is it also atomic-enterprise?